go/go.mod

```diff
@@ -21,9 +23,10 @@ require (
 	github.com/cznic/strutil v0.0.0-20181122101858-275e90344537 // indirect
 	github.com/dgraph-io/badger v2.0.0-rc.2.0.20190625224416-e0016993b790+incompatible
 	github.com/eapache/channels v1.1.0
-	github.com/eapache/queue v1.1.0 // indirect
 	github.com/edsrzf/mmap-go v1.0.0 // indirect
+	github.com/elastic/gosigar v0.10.5 // indirect
 	github.com/etcd-io/bbolt v1.3.3
+	github.com/ethereum/go-ethereum v1.9.6
```

Comment:
Ugh, that repo is huge.

Reply:
Since this is temporary, instead of inflicting the pain of pulling in the entire repo on all of us, just use:
`var defaultRootDerivationPath = []uint32{0x80000000 + 44, 0x80000000 + 60, 0x80000000 + 0, 0}`

Reply:
Will do. This wasn't intended to be merged into master, so no one else should have had to build the entire repo.